addons/binding/org.openhab.binding.blueiris/OSGI-INF/BlueIrisHandlerFactory.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
```

Comment:
It seems that this file has a missing, outdated or otherwise invalid copyright header, can you fix all of them by running 

`mvn license:format` 

Please commit only the ones for your binding.